### PR TITLE
PHPCS: Update configuration and address errors

### PR DESCRIPTION
**Pull request by swissspidy**
## Context

Saw some warnings sneaking in an realized there were a bunch more we were not seeing.

Tested using `vendor/bin/phpcs . --severity=1`

## Summary

See commit messages for details.

### QA

- [x] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1.

### UAT

- [x] UAT should use the same steps as above.

This PR can be tested by following these steps:

1.

## Checklist

- [ ] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #

### CODE REVIEW

File: includes/namespace.php [hunk 1]
```diff
@@ -109,13 +109,20 @@ function new_site( $site ) {
 	if ( ! is_multisite() ) {
 		return;
 	}
+
```
Comment (spacedmonkey) [inline includes/namespace.php:112]: Lots of added new lines in this diff, why?

File: .github/workflows/lint-php.yml [hunk 2]
```diff
@@ -66,7 +66,7 @@ jobs:
         run: composer install --prefer-dist --no-progress --no-interaction
 
       - name: Detect coding standard violations (PHPCS)
-        run: vendor/bin/phpcs -q --report=checkstyle --runtime-set ignore_errors_on_exit 1 --runtime-set ignore_warnings_on_exit 1 | cs2pr --graceful-warnings
+        run: vendor/bin/phpcs -q --report=checkstyle --severity=1 --runtime-set ignore_errors_on_exit 1 --runtime-set ignore_warnings_on_exit 1 | cs2pr --graceful-warnings
```
Comment (spacedmonkey) [inline .github/workflows/lint-php.yml:69]: Why level 1? Could this / should this to be higher?

**Comment by spacedmonkey** [comment]
Do we need to change these? 

https://github.com/google/web-stories-wp/blob/df88e0b9229c821fe5f1fe5584ac009c76799b80/package.json#L214-L215

or 

https://github.com/google/web-stories-wp/blob/df88e0b9229c821fe5f1fe5584ac009c76799b80/composer.json#L103-L104

File: includes/namespace.php (hunk 1 above)
Comment (swissspidy) [reply includes/namespace.php:112]: Figured the code should have some more air to breathe while I was at it.

File: .github/workflows/lint-php.yml (hunk 2 above)
Comment (swissspidy) [reply .github/workflows/lint-php.yml:69]: By default, PHPCS assigns a severity of 5 to all errors and warnings. That means any messages with severity 1-4 are _not_ reported. By specifically adding `severity=1` we see _more_ messages.

If we would make this higher then we would not see any messages at all.

Comment (spacedmonkey) [inline .github/workflows/lint-php.yml:69]: Should this be 
```suggestion
        run: composer phpcs -q --report=checkstyle --runtime-set ignore_errors_on_exit 1 --runtime-set ignore_warnings_on_exit 1 | cs2pr --graceful-warnings
```

Comment (swissspidy) [reply .github/workflows/lint-php.yml:69]: IIRC, we're not using the composer script as there is additional output and differences in exit codes, potentially breaking the `cs2pr` part.

I don't wanna risk doing that.

**Review by spacedmonkey — APPROVED** [review]

**Pull request merged**